lib/tether/ops_linux.go

```diff
@@ -929,6 +935,9 @@ func (t *BaseOperations) Setup(config Config) error {
 		return err
 	}
 
+	// NOTE: ulimit default values should change when we support ulimit configuration
+	setupDefaultULimit()
```

Comment:
This should be in `cmd/tether/ops_linux.go` not in the common `/lib/tether`. default ulimit's in the appliance will still be applied via `/etc/security/limits*`